index.html

```diff
@@ -11,6 +11,8 @@
   </head>
 
   <body>
+    <a href="docs/user_guide/README.md">openapi.auth.yaml</a>
```

Comment:
Is the link text supposed to be "README" or something along those lines?
```suggestion
    <a href="docs/user_guide/README.md">README</a>
```

Reply:
Good catch!

Reply:
Tried to make it all a bit more readable